### ISSUE TITLE
Implement all checkout steps on single page

### ISSUE THREAD

**Issue by artursmet** [mirumee/saleor#670]
According to the design, all checkout steps will be displayed on a single page:

**Comment by anasamoudi** [comment]
Is this planned?

**Comment by patrys** [comment]
Yes but all changes take time to implement. Our current efforts go towards making the dashboard better and easier to use. I think the managing staff deserve this, after all they have to spend there multiple hours a day.
